### PR TITLE
Pair programming with Peter and Cynthia.

### PR DESCRIPTION
**Pull request by Nedramrez**
- Invite collaborators.
- Add the Contact me section skeleton.
- Create a new branch.
- Receive partners' commits.
- Style Contact me section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -340,6 +341,21 @@ <h1>About <br>
...
+            <div class="blue-rounded">
+              <img src="assets/contact-form.svg" alt="Contact form background image">
+              <h2 class="contact-heading">Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+              <form action="https://formspree.io/f/xgedddaq" method="post">
+                <input id="name" type="text" name="name" value="Yeremias"><br>
+                <input id="email" type="email" name="email" value="natajayanj14@gmail.com"><br>
+                <textarea id="message" name="message" cols="50" rows="10" placeholder="Write your message here"></textarea>
+              </form>
+              <button type="button" id="btn">Get in touch</button>
```
Comment (micheaol) [inline index.html]: Your app is not working as expected. When I click on the `submit button` not happens. There are two reasons for this issue:
- The `submit button` should be wrapped in the `form`.
- The `submit button` type should be `submit` and NOT `button`
Kindly fix this issues.

File: index.html [hunk 2]
```diff
@@ -340,6 +341,21 @@ <h1>About <br>
...
+        </main>
+        <section class="contact-me">
+          <footer>
+            <div class="blue-rounded">
+              <img src="assets/contact-form.svg" alt="Contact form background image">
+              <h2 class="contact-heading">Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+              <form action="https://formspree.io/f/xgedddaq" method="post">
+                <input id="name" type="text" name="name" value="Yeremias"><br>
+                <input id="email" type="email" name="email" value="natajayanj14@gmail.com"><br>
```
Comment (micheaol) [inline index.html]: Please ensure that all input fields have the correct associated placeholder text according to the project requirement.

File: index.html [hunk 3]
```diff
@@ -340,6 +341,21 @@ <h1>About <br>
...
-    </main>
+        </main>
+        <section class="contact-me">
+          <footer>
+            <div class="blue-rounded">
+              <img src="assets/contact-form.svg" alt="Contact form background image">
+              <h2 class="contact-heading">Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+              <form action="https://formspree.io/f/xgedddaq" method="post">
+                <input id="name" type="text" name="name" value="Yeremias"><br>
```
Comment (micheaol) [inline index.html]: Please ensure that the name input has the `maxlength="30"` attribute according to the project requirement.

File: index.html [hunk 4]
```diff
@@ -340,6 +341,21 @@ <h1>About <br>
...
+                <input id="email" type="email" name="email" value="natajayanj14@gmail.com"><br>
+                <textarea id="message" name="message" cols="50" rows="10" placeholder="Write your message here"></textarea>
```
Comment (micheaol) [inline index.html]: Please ensure that the message input has the `maxlength="500"` attribute according to the project requirement.

File: index.html [hunk 5]
```diff
@@ -340,6 +341,21 @@ <h1>About <br>
...
         </section>
-    </main>
+        </main>
+        <section class="contact-me">
+          <footer>
+            <div class="blue-rounded">
+              <img src="assets/contact-form.svg" alt="Contact form background image">
+              <h2 class="contact-heading">Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+              <form action="https://formspree.io/f/xgedddaq" method="post">
```
Comment (micheaol) [inline index.html:351]: Please ensure that all input fields have the `required` attribute to activate the validation on submit according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Nedramrez  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by dasileker — APPROVED** [review]
### **Hi @Nedramrez,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nedramrez** [comment]
Thank you @dasileker!

**Pull request merged**